ui/_locales/en/messages.json

```
@@ -823,7 +823,9 @@
       },
       "abilityStateTitle": "Show",
       "abilitiesTypesTitle": "Notify me about abilities type",
-      "accountsTitle": "Show/hide accounts"
+      "accountsTitle": "Show/hide accounts",
+      "accountsReadOnlyInfo": "Read-only accounts can’t see abilities. If you can, we recommend you import the address you want o see abilities for.",
```

```suggestion
      "accountsReadOnlyInfo": "Read-only accounts can’t see abilities. If you can, we recommend you import the address you want to see abilities for.",
```
That "if you can" part sounds weird to me. I would say it should be more like a CTA "You should/can import the address you want to..."

---

@VladUXUI What do you think?

---

I would leave if you can. Because we don't support private key import or other things. So blocking users from seeing a feature, then telling them to do an action that they can't do, just leads to disappointment and messages on discord.
This way we set the expectations a little bit